app/domain/data.ts

```diff
@@ -204,6 +204,10 @@ export const isGeoShapesDimension = (
   return dimension?.__typename === "GeoShapesDimension";
 };
 
+export const isNumericalDimension = (dimension: DimensionMetaDataFragment) => {
+  return dimension.isNumerical;
+};
```

Comment:
Maybe not necessary to have a function here since there is an attribute.

Reply:
Wasn't sure about that, I think you're right it's too much :)